src/adapters/store.js

```diff
@@ -27,8 +35,23 @@ function Store() {
   }
 }
 
+Store.prototype.getUserInfo = async function() {
+  try {
+    if (! (await checkRegistered())) {
```

Comment:
the condition can be reversed to avoid that error prone syntax

Reply:
checkRegistered is not needed anymore with the new logic of calling localStorage until logged into masq